### PR TITLE
Configuration fixes

### PR DESCRIPTION
**Pull request by hluk**
Fixes name of "ssl-verify" option in documentation (this confused me a lot) and "develop" option (which was ignored before).

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -31,7 +31,7 @@ Other possible keys are:
     If specified, this token will be used for authentication. The client
     will not try to obtain any token from the server.
 
-* `ssl_verify`
+* `ssl-verify`
```
Comment (simozhan) [inline README.markdown:34]: Good catch. Yeah. this really confuses the user

File: README.markdown [hunk 2]
```diff
@@ -57,7 +57,7 @@ localhost and a production server. :
         "local": {
             "host": "http://localhost:8000/rest_api/v1/",
             "develop": true,
-            "insecure": false
+            "ssl-verify": false
```
Comment (simozhan) [inline README.markdown:60]: Yeah. As we prefer ssl-verify, we should update this.
In our internal repo, the file pdc_client_config.json should also be updated. But that should be in another patch

**Comment by simozhan** [comment]
LGTM :+1:

**Comment by xiangge** [comment]
Looks good to me. 👍
I think this change is more reasonable.
